string fen_in
---
bool is_success
